Cargo.toml

```diff
@@ -20,7 +20,8 @@ leptos_icons = { version = "0.1", default_features = false, features = [
     "AiStepBackwardFilled",
     "AiFastForwardFilled",
     "AiStepForwardFilled",
-    "BiInfiniteRegular"
+    "BiInfiniteRegular",
+    "RiSwordOthersLine"
```

Comment:
Is there a better way to do this instead of pulling every single Icon in one by one?